extensions/wikia/Insights/specials/UnconvertedInfoboxesPage.class.php

```diff
@@ -95,27 +95,30 @@ public function recache( $limit = false, $ignoreErrors = true ) {
 	 * @return bool|mixed
 	 */
 	public function reallyDoQuery( $limit = false, $offset = false ) {
-		$dbr = wfGetDB( DB_SLAVE, [ $this->getName(), __METHOD__, 'vslow' ] );
-
-		$nonportableTemplates = ( new WikiaSQL() )
-			->SELECT( 'page_title' )->AS_( 'title' )
-			->FROM( 'page' )
-			->WHERE( 'page_namespace' )->EQUAL_TO( NS_TEMPLATE )
-				->AND_( 'page_is_redirect' )->EQUAL_TO( 0 )
```

Comment:
What do you mean? We now use TCS data as a base for the operations so I cannot say we 'skip' it.